src/modules/Builder/features/ActivitySettings/SubscalesConfiguration/SubscaleLookupTable/SubscaleLookupTable.tsx

```diff
@@ -0,0 +1,52 @@
+import { StyledModalWrapper } from 'shared/styles';
+import { Modal, SubmitBtnColor } from 'shared/components';
+
+import { SubscaleLookupTableProps } from './SubscaleLookupTable.types';
+import { getComponent } from './SubscaleLookupTable.utils';
+import { useSubscaleLookupTableSetup } from './SubscaleLookupTable.hooks';
+
+export const SubscaleLookupTable = ({
+  open,
+  subscaleName,
+  tableData,
+  onUpdate,
+  onClose,
+}: SubscaleLookupTableProps) => {
+  const { modalType, step, data, error, setModalType, setError, setStep, onFileReady } =
```

Comment:
pls use  const props =  useSubscaleLookupTableSetup({ tableData });
 const screens = getComponent(props)

Reply:
why? propreties from `useSubscaleLookupTableSetup({ tableData })` distributes attributes not only for `getComponent`. Moreover, there are some properties from SubscaleLookupTable need to add for `getComponent`